documentation/modules/ROOT/partials/modules/all-connectors/ref-connector-monitoring-streaming-metrics.adoc

```diff
@@ -32,11 +32,11 @@
 
 |`Connected`
 |`boolean`
-|Flag that denotes whether the connector is currently connected to the database server.
+|`Flag that denotes whether the connector is currently connected to the database server.
```

Comment:
Could you revert this and the next one below. I had fixed those earlier today.

Reply:
Hmmm. I didn't intentionally insert that single quote.  I removed it in this entry and in the next one. Hope that works. This update will be in the commit I'll do in a moment.